shell/platform/embedder/embedder_thread_host.cc

```diff
@@ -79,13 +83,51 @@ CreateEmbedderTaskRunner(const FlutterTaskRunnerDescription* description) {
                     SAFE_ACCESS(description, identifier, 0u))};
 }
 
+#if WINDOWS
+static void WindowsPlatformThreadConfigSetter(
+    const fml::Thread::ThreadConfig& config) {
+  // set thread name
+  fml::Thread::SetCurrentThreadName(config);
+  // set thread priority
+  switch (config.priority) {
+    case fml::Thread::ThreadPriority::BACKGROUND: {
+      if (SetPriorityClass(GetCurrentProcess(), NORMAL_PRIORITY_CLASS) !=  0) {
+        FML_LOG(ERROR) << "Failed to set IO task runner priority";
+      }
+      break;
+    }
+    case fml::Thread::ThreadPriority::DISPLAY: {
+      if (SetPriorityClass(GetCurrentProcess(), ABOVE_NORMAL_PRIORITY_CLASS) !=  0) {
+        FML_LOG(ERROR) << "Failed to set IO task runner priority";
+      }
+      break;
+    }
+    case fml::Thread::ThreadPriority::RASTER: {
+      if (SetPriorityClass(GetCurrentProcess(), ABOVE_NORMAL_PRIORITY_CLASS) !=  0) {
```

Comment:
Chrome equivalent might be GPU thread: https://source.chromium.org/chromium/chromium/src/+/main:content/gpu/gpu_main.cc;l=234?q=ABOVE_NORMAL_PRIORITY_CLASS&ss=chromium

Uses ABOVE_NORMAL_PRIORITY_CLASS - the flag is enabled by default

Reply:
This isn't correct since this refers to the process priority, which we may need to consider as well